library/Mockery.php

```diff
@@ -92,6 +93,12 @@ public static function mock(...$args)
      */
     public static function spy(...$args)
     {
+        $args = func_get_args();
```

Comment:
This line should not be necessary. ...$args is an array of all function arguments already.